### PR TITLE
[#4939] Supports spring-kafka via option within the kafka plugin

### PR DESCRIPTION
**Pull request by koo-taejin**
Supports spring-kafka via option within the kafka plugin

### CODE REVIEW

File: agent/src/main/resources-local/pinpoint.config [hunk 1]
```diff
@@ -874,6 +874,8 @@ profiler.akka.http.transform.targetparameter=scala.Function1,scala.Function1,akk
 ###########################################################
 profiler.kafka.producer.enable=false
 profiler.kafka.consumer.enable=false
+# Setting when using spring-kafka (In this case, you can leave propeller.kafka.conscher.entryPoint option to empty.)
```
Comment (Xylus) [inline agent/src/main/resources-local/pinpoint.config]: `propeller.kafka.conscher.entryPoint` typo?

File: agent/src/main/resources-release/pinpoint.config [hunk 2]
```diff
@@ -865,6 +865,8 @@ profiler.akka.http.transform.targetparameter=scala.Function1,scala.Function1,akk
 ###########################################################
 profiler.kafka.producer.enable=false
 profiler.kafka.consumer.enable=false
+# Setting when using spring-kafka (In this case, you can leave propeller.kafka.conscher.entryPoint option to empty.)
```
Comment (Xylus) [inline agent/src/main/resources-release/pinpoint.config]: typo as well I think.

File: agent/src/main/resources-local/pinpoint.config (hunk 1 above)
Comment (koo-taejin) [reply agent/src/main/resources-local/pinpoint.config]: Thanks for you're review. I fixed it :)

File: agent/src/main/resources-release/pinpoint.config (hunk 2 above)
Comment (koo-taejin) [reply agent/src/main/resources-release/pinpoint.config]: Thanks for you're review. I fixed it :)

**Pull request merged**